test/modules/linear_boltzmann_solvers/transport_steady/transport_2d_4a_dsa_ortho.lua

```diff
@@ -83,10 +79,9 @@ lbs_block =
       l_max_its = 1000,
       gmres_restart_interval = 30,
       apply_wgdsa = true,
-      wgdsa_l_abs_tol = 1.0e-2,
-    },
-    {
-      groups_from_to = {63, num_groups-1},
+      wgdsa_l_abs_tol = 1.0e-2
+    }, {
```

Comment:
I think this can cause confusion for inexperienced users. I liked the way it was before. Can we encode this in the style/config file?

Reply:
I think the new formatting tool fixes this.